router.go

```diff
@@ -19,14 +23,17 @@ type Options struct {
 	// If the backend is nil, then DefaultRESTConfig, DefaultNamespace, and Scheme are used to create a backend.
 	Backend backend.Backend
 	// If a Backend is provided, then this is ignored. If not provided and needed, then a default is created with Scheme.
-	DefaultRESTConfig *rest.Config
+	RESTConfig *rest.Config
 	// If a Backend is provided, then this is ignored.
-	DefaultNamespace string
+	Namespace string
+	// If a Backend is provided, then this is ignored.
```

Comment:
Duplicate comments

Reply:
These comments are meant to indicate that the individual fields are ignored if a `Backend` is provided. I think the duplicate comments make sense here.